Makefile

```diff
@@ -12,7 +12,6 @@ tools: ## Install all needed tools, e.g. for static checks
 	@echo Installing tools from tools.go
 	@grep '@' req-tools.txt | xargs -tI % go install %
 
-
```

Comment:
Could you, please, explain, why this line was removed?

Reply:
Other code blocks have one blank line between them.

Reply:
Cool, I didn't catch it myself, but it would be better not to mix compliance changes (aka whitespace changes) with functional code changes, so these changes should be moved in separate commit. (first are going compliance/whitespace changes, then functional one).